src/model.ts

```diff
@@ -132,4 +132,35 @@ export class Model {
     }
     return model;
   }
+
+  static modelGetFlatArgs(keyName: string): any[] {
+    return [keyName, 'META', 'BLOB'];
+  }
+
+  static modelRunFlatArgs(modelName: string, inputs: string[], outputs: string[]): any[] {
+    const args: any[] = [modelName, 'INPUTS'];
+    inputs.forEach((value) => args.push(value));
+    args.push('OUTPUTS');
+    outputs.forEach((value) => args.push(value));
+    return args;
+  }
+
+  modelSetFlatArgs(keyName: string) {
+    const args: any[] = [keyName, this.backend.toString(), this.device];
```

Comment:
string[]

Reply:
for this one we need to keep the []any, given that argument of type 'Buffer' is not assignable to parameter of type 'string'